backend/tests/integration/files/test_api_s3.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+import os
+
+import pkg_resources
+import pytest
+from botocore.exceptions import ClientError
+from mock import patch
+
+from inspirehep.files.api import S3
+from inspirehep.files.proxies import s3_instance
+
+BUCKET = "inspire-files-b"
+KEY = "b50c2ea2d26571e0c5a3411e320586289fd715c2"
+
+
+def test_upload_file(base_app, appctx, s3, create_s3_bucket):
+    s3 = S3(s3[0], s3[1])
```

Comment:
use `s3_instance` in all tests instead?

Reply:
I couldn't find a way to do that and pass it the mocked client and resource. It is easier this way